src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/FocusManagerDirectionTests/UnoSamples_Tests.FocusManagerDirection.cs

```diff
@@ -0,0 +1,142 @@
+﻿using NUnit.Framework;
+using SamplesApp.UITests.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.UITest;
+using Uno.UITest.Helpers;
+using Uno.UITest.Helpers.Queries;
+
+namespace SamplesApp.UITests.Windows_UI_Xaml.FocusManagerDirectionTests
+{
+	public partial class UnoSamples_Tests : SampleControlUITestBase
```

Comment:
Forgot about that one. Currently, iOS tests are not run because of #1955. To add those test explicitly for iOS, you'll need to change the class name (call it `FocusManagerDirection_Tests`) and add ` SamplesApp.UITests.Windows_UI_Xaml.FocusManagerDirectionTests` here: https://github.com/unoplatform/uno/blob/master/build/ios-uitest-run.sh#L32-L41. Otherwise, the iOS tests are not run by default.

Reply:
Are the tests really not working on iOS ? I'd rather see those tests running there.

Reply:
Did set up to run on IOS. Added everything as mentioned by Jerome.  But still not sure how exactly it will work on IOS. Jerome is busy so for now just giving proper setup for IOS and will see later.